docs/index.asciidoc

```diff
@@ -130,6 +131,21 @@ SSL key path
 
 SSL key passphrase
 
+[id="plugins-{type}s-{plugin}-ssl_supported_protocols"]
+===== `ssl_supported_protocols`
+
+  * Value type is <<string,string>>
+  * Allowed values are: `'TLSv1.1'`, `'TLSv1.2'`, `'TLSv1.3'`
+  * Default depends on the JDK being used. With up-to-date Logstash, the default is `['TLSv1.2', 'TLSv1.3']`.
+    `'TLSv1.1'` is not considered secure and is only provided for legacy applications.
+
+List of allowed SSL/TLS versions to use when establishing a secure connection.
+The plugin has a limitation and can only enable a range of protocols, e.g. setting `['TLSv1.3', 'TLSv1.1']` does not disable `TLSv1.2`.
```

Comment:
Is there a more strategic reason for this limitiation? otherwise we can explicitly disable versions the user doesn't list, e.g:

```ruby
      if !ssl_supported_protocols.include?("TLSv1.2")
        @ssl_context.options = @ssl_context.options | OpenSSL::SSL::OP_NO_TLSv1_2
      end
```

Reply:
that would certainly work - copied the `ssl_supported_protocols` from elsewhere where the impl existed and was acceptable given previously users could only set `tls_min_version` and `tls_max_version`, in this case we could implement the special case for the 1.2 "hole" in the interval as suggested, thanks.